deploy/olm-catalog/qdrouterd-operator/0.1.0/interconnectedcloud.package.yaml

```diff
@@ -0,0 +1,5 @@
+#! package-manifest: deploy/catalog_resources/redhat/businessautomation-operator.v1.0.0.clusterserviceversion.yaml
```

Comment:
package-manifest needs to be updated as above

Reply:
Yes, and I've update and pushed.